src/Simplic.Icon.UI/ViewModel/IconEditorViewModel.cs

```diff
@@ -210,7 +220,28 @@ public ObservableCollection<IconViewModel> Icons
         public IconViewModel SelectedIcon
         {
             get { return selectedIcon; }
-            set { PropertySetter(value, (newValue) => { selectedIcon = newValue; }); }
+            set
+            {
+                PropertySetter(value, (newValue) => { selectedIcon = newValue; });
+                RaisePropertyChanged(nameof(IsSelectedIcon));
+            }
+        }
+
+        public Visibility IsSelectedIcon
+        {
+            get
+            {
+                if (selectedIcon != null)
+                {
+                    isSelectedIcon = Visibility.Visible;
+                }
+                else
+                {
+                    isSelectedIcon = Visibility.Hidden;
+                }
+                return isSelectedIcon;
+            }
+            set { PropertySetter(value, (newValue) => { isSelectedIcon = newValue; }); }
```

Comment:
Is this used anywhere? Also the naming of the variable does not really tell what the value should represent. And missing comment